eos-updater/com.endlessm.Updater.xml

```diff
@@ -2,49 +2,297 @@
+
+      Description body of the OSTree commit available as an update, or the empty
+      string if it’s not set or if no update is available. This is the
+      description of the update.
+    -->
+    <property name="UpdateMessage" type="s" access="read"/>
+
+    <!--
+      Version:
+
+      Version number of the OSTree commit available as an update, or the empty
+      string if it’s not set or if no update is available.
+    -->
+    <property name="Version" type="s" access="read"/>
+
+    <!--
+      DownloadSize:
+
+      Size (in bytes) of the update when downloaded, or `-1` if an update is
+      available but its download size is unknown. `0` if no update is available.
+    -->
+    <property name="DownloadSize" type="x" access="read"/>
+
+    <!--
+      DownloadedBytes:
+
+      Number of bytes of the update which have already been downloaded. This
+      will be `0` before a download starts, and could be `-1` if the
+      `DownloadSize` is unknown.
+    -->
+    <property name="DownloadedBytes" type="x" access="read"/>
+
+    <!--
+      UnpackedSize:
+
+      Size (in bytes) of the update when unpacked, or `-1` if an update is
+      available but its unpacked size is unknown. `0` if no update is available.
+    -->
+    <property name="UnpackedSize" type="x" access="read"/>
+
+    <!--
+      FullDownloadSize:
+
+      Version of `DownloadSize` which also includes the sizes of parts of the
+      update which are already present locally (and hence which don’t need to
+      be downloaded again).
+    -->
     <property name="FullDownloadSize" type="x" access="read"/>
+
+    <!--
+      FullUnpackedSize:
+
+      Version of `UnpackedSize` which also includes the sizes of parts of the
+      update which are already unpacked locally (and hence which won’t occupy
+      further disk space once the update is applied).
+    -->
     <property name="FullUnpackedSize" type="x" access="read"/>
-    <!-- An error code, in an unspecified error domain! See ErrorName. -->
-    <property name="ErrorCode"        type="u" access="read"/>
-    <!-- a fully-qualified D-Bus error name, as might be returned from a D-Bus
-         method. "com.endlessm.Updater.Error.LiveBoot" when the system is a
-         read-only live USB, where updates are not supported.
-      -->
-    <property name="ErrorName"        type="s" access="read"/>
-    <!-- a human-readable, albeit unlocalized, error message -->
-    <property name="ErrorMessage"     type="s" access="read"/>
 
+    <!--
+      ErrorCode:
+
+      Error code of the current error, or `0` if no error has been reported.
+      This is in an unspecified error doman, and hence is useless.
+    -->
+    <property name="ErrorCode" type="u" access="read"/>
+
+    <!--
+      ErrorName:
+
+      A fully-qualified D-Bus error name, as might be returned from a D-Bus
+      method. `com.endlessm.Updater.Error.LiveBoot` when the system is a
+      read-only live USB, where updates are not supported.
```

Comment:
I think this sentence is redundant with the list below.

Reply:
:ok: